webapp/src/auth.ts

```diff
@@ -48,6 +50,7 @@ export type UserProfile = {
 export type UserPreferences = {
     language?: string;
     highContrast?: boolean;
+    reader?: string;
 }
 
 const DEFAULT_USER_PREFERENCES: () => UserPreferences = () => ({
```

Comment:
Is it worth adding a default value here for the reader? Maybe in empty string?

Reply:
hahaha yeah i'll do that!